Sming/Makefile-project.mk

```diff
@@ -143,7 +143,7 @@ ifneq ($(WIFI_PWD), "")
 endif
 
 # linker flags used to generate the main object file
-LDFLAGS		= -nostdlib -u call_user_start -Wl,-static -Wl,--gc-sections
+LDFLAGS		= -nostdlib -u call_user_start -Wl,-static -Wl,--gc-sections -Wl,-Map=$(FW_BASE)/firmware.map
```

Comment:
this is very useful to generate mapfile where one can see what functions got linked and their sizes